cmd/kubeadm/app/util/staticpod/utils_test.go

```diff
@@ -630,6 +630,35 @@ spec:
   - image: gcr.io/google_containers/etcd-amd64:3.1.11
 status: {}
 `
+	validPodWithDifferentFieldsOrder = `
+apiVersion: v1
+kind: Pod
+metadata:
+  labels:
+    tier: control-plane
+    component: etcd
+  name: etcd
+  namespace: kube-system
+spec:
+  containers:
+  - image: gcr.io/google_containers/etcd-amd64:3.1.11
```

Comment:
value  of the image is not equal to other

Reply:
IMO, the manifest is just for test purpose, so the value is irrelevant.